java/cdk8s-core/pom.xml

```diff
@@ -26,5 +26,44 @@
   <build>
     <sourceDirectory>src/main/java</sourceDirectory>
     <testSourceDirectory>src/test/java</testSourceDirectory>
+
+    <plugins>
```

Comment:
I understand you're trying to be more java idiomatic here I guess, but lets talk about this because we've never added something like this this before and its not in our init templates. 

If we decide to add it, I think it should be added to the templates, and then updated here. Otherwise it just creates a big divergence and might be confusing.